components/supervisor/frontend/src/index.ts

```diff
@@ -10,9 +10,37 @@
  */
 import { IDEMetricsServiceClient, MetricsName } from "./ide/ide-metrics-service-client";
 IDEMetricsServiceClient.addCounter(MetricsName.SupervisorFrontendClientTotal).catch(() => {})
-window.addEventListener('error', () => {
-    IDEMetricsServiceClient.addCounter(MetricsName.SupervisorFrontendErrorTotal).catch(() => {})
-})
+
+declare global {
+    interface Window {
+        gatheredErrors: Map<string, string>[];
+    }
+}
+
+window.addEventListener('error', (event) => {
+
+    const flags = new Map();
+
+    //@ts-ignore We take a look at what is the resource that was attempted to load;
+    const resourceSource = event?.target?.getAttribute('src') || event?.target?.getAttribute('href');
+
+    // If the event has a `target`, it means that it wasn't a script error
+    if (resourceSource) {
+        flags.set('resource', 'vscode-web-workbench');
```

Comment:
it is only the case if resourceSource is matching to `https://ide.gitpod.io/blobserve/eu.gcr.io/gitpod-core-dev/build/ide/code:commit`, otherwise resource should be `Unknown`

Reply:
please not use it as i give it to you, but test, the url can be relative so maybe you need to match for substring like `ide/code:commit-`

It should work also in SH, but we can test it separately.

Reply:
Added a regular expression to match this.

Reply:
hm, i don't see any regular expression 🤔

Reply:
Git did something weird, going to re-push this change.

Reply:
Donezos